common/src/main/java/net/adoptopenjdk/icedteaweb/http/HttpUtils.java

```diff
@@ -34,39 +34,40 @@
  obligated to do so.  If you do not wish to do so, delete this
  exception statement from your version.
  */
-package net.sourceforge.jnlp.util;
-
-import net.sourceforge.jnlp.util.logging.OutputController;
+package net.adoptopenjdk.icedteaweb.http;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.HttpURLConnection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class HttpUtils {
+    private final static Logger LOG = LoggerFactory.getLogger(HttpUtils.class);
 
     /**
      * Ensure a HttpURLConnection is fully read, required for correct behavior.
      * Captured IOException is consumed and printed
-     * @param c the connection to be closed silently
+     * @param c the connection to be closed silentlyboolean isHeadRequest = request.equals(
```

Comment:
I think here a find&replace destroyed the doc ;)

Reply:
jep! thx ;-)